test/conformance/service_test.go

```diff
@@ -396,6 +398,20 @@ func TestReleaseService(t *testing.T) {
 		[]string{expectedFirstRev, expectedSecondRev},
 		[]string{"latest", "candidate", "current"},
 		[]string{expectedThirdRev, expectedSecondRev, expectedFirstRev})
+
+	// Now update the service to use `@latest` as candidate.
+	revisions[1] = v1alpha1.ReleaseLatestRevisionKeyword
+	logger.Info("Updating Service to split traffic between two `current` and `@latest`")
```

Comment:
Your combining of traffic target names with revision keywords confuses me.

```suggestion
	logger.Info("Updating Service to split traffic between the second revision and `@latest`")
```